docs/customize/configure.md

```diff
@@ -0,0 +1,621 @@
+
+## Celery Configuration
+
+### CELERY_BROKER_URL
+
+URL for the Celery message broker (default: RabbitMQ).
+
+### CELERY_RESULT_BACKEND
+
+URL for the Celery result backend (default: Redis).
+
+### CELERY_BEAT_SCHEDULE
+
+Scheduled tasks configuration for Celery (cronjobs).
+
+Example:
+
+```python
+CELERY_BEAT_SCHEDULE = {
+    "indexer": {
+        "task": "invenio_indexer.tasks.process_bulk_queue",
+        "schedule": timedelta(minutes=5),
+    },
+    "send_overdue_loan_reminders": {
+        "task": "invenio_app_ils.circulation.notifications.tasks.send_overdue_loans_notification_reminder",
+        "schedule": crontab(minute=0, hour=2),
+    },
+    "stats-aggregate-events": {
+        "task": "invenio_stats.tasks.aggregate_events",
+        "schedule": timedelta(hours=3),
+        "args": [("record-view-agg", "file-download-agg")],
+    },
+}
+```
+
+## Feature Configurations
+
+### EXTEND_LOANS_LOCATION_UPDATED (True/False)
+
+Flag to indicate whether to start a task to update active loans when closures of a location are updated.
+
+### EXTEND_LOANS_SCHEDULE_TIME
+
+Time at which the task to extend loans should be run.
+
+```python
+EXTEND_LOANS_SCHEDULE_TIME = datetime.time(2)
+```
+
+### ILS_SELF_CHECKOUT_ENABLED (True/False)
+
+Enable or disable Self-Checkout feature endpoint.
+
+## Database Configuration
+
+### SQLALCHEMY_DATABASE_URI
+
+Database URI including user and password.
+
+```python
+SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://test:psw@localhost/ils"
+```
+
+## JSONSchemas Configuration
+
+### JSONSCHEMAS_HOST
+
+Hostname used in URLs for local JSONSchemas.
+
+```python
+JSONSCHEMAS_HOST = "127.0.0.1:5000"
+```
+
+## CSRF Configuration
+
+### REST_CSRF_ENABLED (True/False)
+
+Enable or disable CSRF protection for REST APIs.
+
+## CORS Configuration
+
+### REST_ENABLE_CORS (True/False)
+
+Enable or disable CORS support for REST APIs.
+
+### CORS_SEND_WILDCARD (True/False)
+
+Allow or disallow credentials to be sent with CORS requests.
+
+### CORS_SUPPORTS_CREDENTIALS (True/False)
+
+Exable or disable supporting credentials for CORS requests.
+
+## Flask Configuration
+
+### SECRET_KEY
+
+Secret key used for session management and CSRF protection.
+
+### MAX_CONTENT_LENGTH
+
+Maximum allowed size for uploaded files via form data.
+
+```python
+MAX_CONTENT_LENGTH = 100 * 1024 * 1024  # 100 MiB
+```
+
+### SESSION_COOKIE_SECURE (True/False)
+
+Set cookie with the secure flag by default.
+
+### SESSION_COOKIE_SAMESITE
+
+Set SameSite attribute for session cookies.
+
+```python
+SESSION_COOKIE_SAMESITE = "Lax"
+```
+
+### APP_ALLOWED_HOSTS
+
+List of allowed hosts for the application.
+
+```python
+APP_ALLOWED_HOSTS = ["localhost", "127.0.0.1"]
+```
+
+### SPA_HOST
+
+Host URL for Single Page Application.
+
+```python
+SPA_HOST = "https://127.0.0.1:3000"
+```
+
+### SPA_PATHS
+
+Paths and placeholders for Single Page Application routes.
+
+```python
+SPA_PATHS = dict(
```

Comment:
I would drop this one from docs, I am not sure what will happen if it is modified